### ISSUE TITLE
Runtime error during local development

### ISSUE THREAD

**Issue by dmwelgus** [looker-open-source/looker-explore-assistant#43]
I'm seeing the following runtime error when I try to run the extension locally:

<img width="1148" alt="Screenshot 2024-06-05 at 5 01 46 PM" src="https://github.com/looker-open-source/looker-explore-assistant/assets/5125058/690f77e9-fcee-4854-bd55-7cc96753fe7b">

I think it might be an issue with how the examples were loaded to Big Query. Is `explore_assistant_examples` supposed to have a separate row for each example (i.e. {'input': ..., 'output': ...}) or is it supposed to have a single row with the examples packed into an array (e.g. [{'input': ..., 'output': ...}, {'input': ..., 'output': ...}]).

**Comment by dmwelgus** [comment]
Nevermind. The issue was that I didn't have any examples loaded into `explore_assistant_refinement_examples`. It's working now

**Comment by arodriguezd** [comment]
Hi, I have the same problem, I have deployed a GCP instance for the frontend and we are using BigQuery as backend the backend deployment was made with the example found in the repository. 
All in the same project. The Bigquery API has been enabled at the instance level. On the other hand the nginx has been installed, to perform proxy with the application. 

Also take down the examples in the bigquery.


charge the capture of one of the examples. 
![image](https://github.com/looker-open-source/looker-explore-assistant/assets/21213875/e2dc7cc2-1838-4ae1-8e5b-de7e12da48b8)


![image](https://github.com/looker-open-source/looker-explore-assistant/assets/21213875/efdc55ab-c9a1-48a7-8e53-992a92df4d13)
Thanks in advance for any help.

**Comment by cavvia** [comment]
I have the same error with the BigQuery Backend... in my case both tables have examples loaded into them successfully (via the script) and the BQ connection works fine from Looker. I have verified the data is present in BigQuery Studio. On the BigQuery side I can see some queries coming in and I see the following errors in the logs:

```
Syntax error: SELECT list must not be empty
```

I can see the select statement in the js and it does look fine though (`SELECT examples FROM...`). Could be an escaping/encoding issue with the query string perhaps, which has newline chars encoded into it.

**Comment by cavvia** [comment]
I also got this error with the BigQuery backend. In my case I verified via BigQuery Studio that both example tables are loaded with data (using the script provided). The BigQuery connection was also working fine. 

On the BigQuery side, I can see requests coming in via the logging console but no errors. However, when looking at the query traces, I could see the query using a `WHERE explore="model:explore"` syntax in its constraint. This was unexpected and should be documented in the examples script documentation. Those scripts take a `--explore_id` arg which should actually be populated with a `model:explore` syntax in order for this to work. After re-running those scripts with the correct `explore_id` the error cleared.
